sdks/java/build-tools/src/main/resources/beam/findbugs-filter.xml

```diff
@@ -1,6 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <FindBugsFilter>
   <Bug category="I18N"/>
+  <Bug pattern="DM_STRING_CTOR"/>
```

Comment:
It emits a bunch of warnings on normal String concatenation, for example this block from [CoderRegistry.java](https://github.com/apache/incubator-beam/blob/master/sdks/java/core/src/main/java/org/apache/beam/sdk/coders/CoderRegistry.java#L245):

```java
    if (paramCoderOrNull != null) {
      return paramCoderOrNull;
    } else {
      // DM_STRING_CTOR below
      throw new CannotProvideCoderException(
          "Cannot infer coder for type parameter " + param.getName());
    }
```

I suspect perhaps they're analyzing compiled symbols and can't tell the difference? Googling around I haven't found others complaining about it, which is surprising.